public/v4/apps/openspeedtest.yml

```diff
@@ -2,23 +2,23 @@ captainVersion: 4
 
 services:
     '$$cap_appname':
-        image: index.docker.io/openspeedtest/latest:$$cap_openspeedtest_version
+        image: index.docker.io/test/latest:$$cap_test_version
```

Comment:
lots of wrong replacement here. All "openspeedtest" instances in the definition are converted to test